common/test/acceptance/pages/lms/course_page.py

```diff
@@ -38,3 +39,14 @@ def has_tab(self, tab_name):
         """
         tab_nav = TabNavPage(self.browser)
         return tab_name in tab_nav.tab_names
+
+    def skip_to_main_content(self):
+        """
+        Ensures skip links function and focus on the intended href
+        """
+
+        skip_to = self.q(css=".nav-skip").attrs('href')[0]
+        self.q(css=".nav-skip").first().click()
+        return EmptyPromise(
+            lambda: self.q(skip_to).present, "Main content area received focus"
```

Comment:
I haven't tested this, but I think you should be able to just be able to check for `self.q("{}:focus".format(skip_to)).present` instead?